pkg/mqtrigger/messageQueue/kafka/kafka.go

```diff
@@ -201,8 +201,9 @@ func (kafka Kafka) getTLSConfig() (*tls.Config, error) {
 
 	tlsConfig.Certificates = []tls.Certificate{cert}
 
-	if err != nil {
-		return nil, err
+	skipVerify, err := strconv.ParseBool(os.Getenv("INSECURE_SKIP_VERIFY"))
+	if err != nil && skipVerify {
```

Comment:
Something seems odd here - so if err is not nil and assuming skipVerify is true then set the value. But also no separate handling of error per se. Should we split error handling - as in if there is an error, then return and then set flag separately?

Reply:
There was problem in the logic. I have fixed the issue.